app/views/smart_proxies/welcome.html.erb

```diff
@@ -0,0 +1,14 @@
+<% content_for(:title, _("Smart Proxies")) %>
+<div class="blank-slate-pf">
+  <div class="blank-slate-pf-icon">
+    <%= icon_text("sitemap", "", :kind => "fa") %>
+  </div>
+  <h1><%= _('Smart Proxies') %></h1>
+  <p><%= _("The Smart Proxy is a project which provides a RESTful API to various subsystems.
+           The Smart Proxy provides an easy way to add or extended existing subsystems, via
+           configurable DHCP (ISC DHCP and MS DHCP Servers), DNS (Bind and MS DNS Servers), Puppet, etc.") %></p>
```

Comment:
1. Remove listing of supported providers, this will become out of date.
1. Unsure what "configurable DHCP, DNS" means?